hack/tools/release/notes.go

```diff
@@ -452,27 +452,32 @@ func generateReleaseNoteEntry(c *commit) (*releaseNoteEntry, error) {
 	switch {
 	case strings.HasPrefix(entry.title, ":sparkles:"), strings.HasPrefix(entry.title, "✨"):
 		entry.section = features
-		entry.title = strings.TrimPrefix(entry.title, ":sparkles:")
-		entry.title = strings.TrimPrefix(entry.title, "✨")
+		entry.title = strings.TrimLeft(strings.TrimPrefix(entry.title, ":sparkles:"), " ")
```

Comment:
Can we move the title calculation to a func and then just call it in all relevant places? The current implementation is a lot of copy & paste (already was before this PR)